integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/wait/service/environment/EnvironmentWaitObject.java

```diff
@@ -82,6 +82,11 @@ public boolean isDeleted() {
         return environment.getEnvironmentStatus().equals(ARCHIVED);
     }
 
+    @Override
+    public boolean isFailedButIgnored() {
+        return false;
```

Comment:
So in case of environment we cannot ignore failed states.

Reply:
Yes, this is step 1 to fix this problem, just the concrete tests. I then can create a 2nd step, after tests are green, to allow it for every WaitObject.
I created a jira for that: https://jira.cloudera.com/browse/CB-14926